bot/cogs/games/games.py

```diff
@@ -39,3 +40,31 @@ async def ball8(self, ctx: Context, *, question: str) -> None:
         embed.add_field(name="Answer", value=answer)
 
         await ctx.send(embed=embed)
+
+    @command(aliases=["pokesearch"])
+    async def pokemon(self, ctx: Context, pokemon: str) -> None:
+        """
+        Fetches data about a given pokemon eg. pokemon pikachu.
```

Comment:
It would look better if this was wrapped in an inline codeblock since this is a reference to the command and this will provide a visual separation to it
```suggestion
        Fetches data about a given pokemon eg. `pokemon pikachu`.
```